### ISSUE TITLE
Crash on first start with v1.4.2 on Android 5.1.1: FATAL EXCEPTION: main

### ISSUE THREAD

**Issue by stuart12** [nextcloud/android#1517]
### Actual behaviour
- Nextcloud crashes on first start

### Expected behaviour
- Nextcloud should ask me to enter the server url, username and password
 
### Steps to reproduce
1. Install and use Owncloud 
3. Install Nextcloud 1.4.2 from F-Droid
4. run Nextcloud
5. read "Unfortunately, nextcloud has stopped"

### Environment data
Android version: Android 5.1.1 (cyanogenmod 12.1-20160822-NIGHTLY-scorpion_windy)

Device model: Xperia Z3 Tablet Compact Wifi (SGP611) scorpion_windy

Stock or customized system: cyanogenmod

Nextcloud app version: 1.4.2 (versionCode=10040299 targetSdk=25) from F-Droid

Nextcloud server version: crashes before entering the server URL

#### Nextcloud log (data/nextcloud.log)
```
I/Timeline( 1469): Timeline: Activity_launch_request id:com.nextcloud.client time:1014317
I/ActivityManager(  790): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[960,650][1160,797] (has extras)} from uid 10016 on display 0
V/WindowManager(  790): addAppToken: AppWindowToken{1620b4f3 token=Token{2b151b62 ActivityRecord{1c06a2d u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t4495}}} to stack=1 task=4495 at 0
V/WindowManager(  790): Adding window Window{1f4ab4ba u0 Starting com.nextcloud.client} at 2 of 7 (after Window{35258dd0 u0 com.cyanogenmod.trebuchet/com.android.launcher3.Launcher})
W/BroadcastQueue(  790): Permission Denial: broadcasting Intent { act=com.android.launcher3.action.LAUNCH flg=0x10 (has extras) } from com.cyanogenmod.trebuchet (pid=1469, uid=10016) requires com.google.android.launcher.permission.RECEIVE_LAUNCH_BROADCASTS due to receiver com.google.android.googlequicksearchbox/com.google.android.search.core.icingsync.ApplicationLaunchReceiver
W/BroadcastQueue(  790): Permission Denial: receiving Intent { act=com.android.launcher3.action.LAUNCH flg=0x10 (has extras) } to com.google.android.googlequicksearchbox/com.google.android.search.core.icingsync.ApplicationLaunchReceiver requires com.android.launcher3.permission.RECEIVE_LAUNCH_BROADCASTS due to sender com.cyanogenmod.trebuchet (uid 10016)
W/BroadcastQueue(  790): Permission Denial: broadcasting Intent { act=com.android.launcher3.action.LAUNCH flg=0x10 (has extras) } from com.cyanogenmod.trebuchet (pid=1469, uid=10016) requires com.google.android.launcher.permission.RECEIVE_LAUNCH_BROADCASTS due to receiver com.google.android.gms/.icing.proxy.ApplicationLauncherReceiver
W/BroadcastQueue(  790): Permission Denial: receiving Intent { act=com.android.launcher3.action.LAUNCH flg=0x10 (has extras) } to com.google.android.gms/.icing.proxy.ApplicationLauncherReceiver requires com.android.launcher3.permission.RECEIVE_LAUNCH_BROADCASTS due to sender com.cyanogenmod.trebuchet (uid 10016)
W/BroadcastQueue(  790): Permission Denial: receiving Intent { act=com.android.launcher3.action.LAUNCH flg=0x10 (has extras) } to com.google.android.gms/.chimera.GmsIntentOperationService$GmsExternalReceiver requires com.android.launcher3.permission.RECEIVE_LAUNCH_BROADCASTS due to sender com.cyanogenmod.trebuchet (uid 10016)
D/ZenLog  (  790): intercepted: 0|android|17039406|null|1000,!priority
V/NotificationService(  790): pkg=android canInterrupt=false intercept=true
V/FileDisplayActivity( 2234): onCreate() start
W/art     ( 2234): Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
D/FileDisplayActivity( 2234): onCreate(Bundle) starting
I/AppCompatViewInflater( 2234): app:theme is now deprecated. Please move to using android:theme instead.
V/FileDisplayActivity( 2234): onCreate() end
D/FileDisplayActivity( 2234): onStart() starting
D/ExternalLinks( 2234): links disabled
E/FileDisplayActivity( 2234): Access to unexisting list of files fragment!!
E/FileDisplayActivity( 2234): Access to unexisting list of files fragment!!
E/FileDisplayActivity( 2234): OCFileListFragment is null
V/FileDisplayActivity( 2234): onResume() start
D/FileDisplayActivity( 2234): onResume() starting
I/OCFileListFragment( 2234): onAttach
I/OCFileListFragment( 2234): onCreateView() start
D/ExtendedListFragment( 2234): onCreateView
I/OCFileListFragment( 2234): onCreateView() end
I/OCFileListFragment( 2234): onActivityCreated() start
V/NotificationService(  790): pkg=de.mangelow.throughput canInterrupt=false intercept=true
V/FileDisplayActivity( 2234): onResume() end
D/OpenGLRenderer( 2234): Use EGL_SWAP_BEHAVIOR_PRESERVED: true
D/Atlas   ( 2234): Validating map...
V/WindowManager(  790): Adding window Window{245206c5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity} at 2 of 8 (before Window{1f4ab4ba u0 Starting com.nextcloud.client})
D/OperationsService( 2234): Creating service
I/AccountAuthenticator( 2234): Adding account with type nextcloud and auth token null
D/FileDownloader( 2234): Creating service
I/Timeline( 2234): Timeline: Activity_launch_request id:com.nextcloud.client time:1014698
I/ActivityManager(  790): START u0 {flg=0x10800004 cmp=com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity (has extras)} from uid 10181 on display 0
V/WindowManager(  790): addAppToken: AppWindowToken{202c9d28 token=Token{243d5f4b ActivityRecord{365b181a u0 com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity t4495}}} to stack=1 task=4495 at 1
V/WindowManager(  790): Adding window Window{1f4ab4ba u0 Starting com.nextcloud.client} at 3 of 8 (after Window{245206c5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity})
D/FileUploader( 2234): Creating service
V/UploadsStorageManager( 2234): Updating state of any killed upload
E/SQLiteLog( 2234): (284) automatic index on list_of_uploads(status)
V/UploadsStorageManager( 2234): No upload was killed
D/FileObserverService( 2234): Starting command Intent { act=com.owncloud.android.services.observer.FileObserverService.action.START_OBSERVATION cmp=com.nextcloud.client/com.owncloud.android.services.observer.FileObserverService }
D/FileObserverService( 2234): Loading all kept-in-sync files from database to start watching them
I/Adreno-EGL( 2234): <qeglDrvAPI_eglInitialize:410>: EGL 1.4 QUALCOMM build: AU_LINUX_ANDROID_LA.BF.1.1.1_RB1.05.01.00.042.030_msm8974_LA.BF.1.1.1_RB1__release_AU ()
I/Adreno-EGL( 2234): OpenGL ES Shader Compiler Version: E031.25.03.06
I/Adreno-EGL( 2234): Build Date: 07/13/15 Mon
I/Adreno-EGL( 2234): Local Branch: mybranch11906725
I/Adreno-EGL( 2234): Remote Branch: quic/LA.BF.1.1.1_rb1.26
I/Adreno-EGL( 2234): Local Patches: NONE
I/Adreno-EGL( 2234): Reconstruct Branch: AU_LINUX_ANDROID_LA.BF.1.1.1_RB1.05.01.00.042.030 + 6151be1 + a1e0343 + 002d7d6 + 7d0e3f7 +  NOTHING
I/OpenGLRenderer( 2234): Initialized EGL, version 1.4
D/OpenGLRenderer( 2234): Enabling debug mode 0
D/FileActivity( 2234): Operations service connected
V/FileDisplayActivity( 2234): onPause() start
D/FileDisplayActivity( 2234): onPause() ending
V/FileDisplayActivity( 2234): onPause() end
D/FileDisplayActivity( 2234): Download service connected
D/FileDisplayActivity( 2234): Upload service connected
D/AuthenticatorActivity( 2234): onCreate(Bundle) starting
I/Timeline( 2234): Timeline: Activity_launch_request id:com.nextcloud.client time:1014871
I/ActivityManager(  790): START u0 {cmp=com.nextcloud.client/com.owncloud.android.ui.activity.WhatsNewActivity} from uid 10181 on display 0
V/WindowManager(  790): addAppToken: AppWindowToken{134be940 token=Token{17c86cc3 ActivityRecord{3d4c9072 u0 com.nextcloud.client/com.owncloud.android.ui.activity.WhatsNewActivity t4495}}} to stack=1 task=4495 at 2
V/WindowManager(  790): Adding window Window{1f4ab4ba u0 Starting com.nextcloud.client} at 3 of 8 (after Window{245206c5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity})
I/TextInputLayout( 2234): EditText added is not a TextInputEditText. Please switch to using that class instead.
I/TextInputLayout( 2234): EditText added is not a TextInputEditText. Please switch to using that class instead.
I/TextInputLayout( 2234): EditText added is not a TextInputEditText. Please switch to using that class instead.
I/TextInputLayout( 2234): EditText added is not a TextInputEditText. Please switch to using that class instead.
I/TextInputLayout( 2234): EditText added is not a TextInputEditText. Please switch to using that class instead.
D/AuthenticatorActivity( 2234): onStart() starting
D/AuthenticatorActivity( 2234): onResume() starting
V/WindowManager(  790): Adding window Window{3ea6596c u0 com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity} at 3 of 9 (before Window{1f4ab4ba u0 Starting com.nextcloud.client})
D/AuthenticatorActivity( 2234): onPause() ending
D/WhatsNewActivity( 2234): onCreate(Bundle) starting
W/ResourceType( 2234): Failure getting entry for 0x7f08000a (t=7 e=10) (error -75)
D/AndroidRuntime( 2234): Shutting down VM
--------- beginning of crash
E/AndroidRuntime( 2234): FATAL EXCEPTION: main
E/AndroidRuntime( 2234): Process: com.nextcloud.client, PID: 2234
E/AndroidRuntime( 2234): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.activity.WhatsNewActivity}: android.content.res.Resources$NotFoundException: String resource ID #0x7f08000a
E/AndroidRuntime( 2234): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2358)
E/AndroidRuntime( 2234): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2420)
E/AndroidRuntime( 2234): 	at android.app.ActivityThread.access$900(ActivityThread.java:154)
E/AndroidRuntime( 2234): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
E/AndroidRuntime( 2234): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime( 2234): 	at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime( 2234): 	at android.app.ActivityThread.main(ActivityThread.java:5294)
E/AndroidRuntime( 2234): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 2234): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime( 2234): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
E/AndroidRuntime( 2234): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
E/AndroidRuntime( 2234): Caused by: android.content.res.Resources$NotFoundException: String resource ID #0x7f08000a
E/AndroidRuntime( 2234): 	at android.content.res.Resources.getText(Resources.java:321)
E/AndroidRuntime( 2234): 	at android.widget.TextView.setText(TextView.java:4135)
E/AndroidRuntime( 2234): 	at com.owncloud.android.ui.activity.WhatsNewActivity.onCreate(WhatsNewActivity.java:142)
E/AndroidRuntime( 2234): 	at android.app.Activity.performCreate(Activity.java:5990)
E/AndroidRuntime( 2234): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
E/AndroidRuntime( 2234): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2311)
E/AndroidRuntime( 2234): 	... 10 more
W/ActivityManager(  790):   Force finishing activity 1 com.nextcloud.client/com.owncloud.android.ui.activity.WhatsNewActivity
W/ActivityManager(  790):   Force finishing activity 2 com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity
I/Adreno-EGL(  790): <qeglDrvAPI_eglInitialize:410>: EGL 1.4 QUALCOMM build: AU_LINUX_ANDROID_LA.BF.1.1.1_RB1.05.01.00.042.030_msm8974_LA.BF.1.1.1_RB1__release_AU ()
I/Adreno-EGL(  790): OpenGL ES Shader Compiler Version: E031.25.03.06
I/Adreno-EGL(  790): Build Date: 07/13/15 Mon
I/Adreno-EGL(  790): Local Branch: mybranch11906725
I/Adreno-EGL(  790): Remote Branch: quic/LA.BF.1.1.1_rb1.26
I/Adreno-EGL(  790): Local Patches: NONE
I/Adreno-EGL(  790): Reconstruct Branch: AU_LINUX_ANDROID_LA.BF.1.1.1_RB1.05.01.00.042.030 + 6151be1 + a1e0343 + 002d7d6 + 7d0e3f7 +  NOTHING
I/OpenGLRenderer(  790): Initialized EGL, version 1.4
E/Diag_Lib(  312): linux_qmi_qmux_io.c: Unable to open port id /dev/smdcntl0, error [110:Connection timed out]
E/Diag_Lib(  312): qmi_qmux_open_connection: QMI_QMUX_IO_PLATFORM_OPEN_CONN failed -1 
E/Diag_Lib(  312): qmuxd:  opening connection for RMNET0 failed on attempt 48, Waiting 500 milliseconds before retry
W/UdcContextManagerHelper( 1406): Empty context buffer. Thus might mean that the context is not synced down.
W/UploadOptInStatusOp( 1406): Empty context model while retrieving upload opt-in status!
I/Timeline(  790): Timeline: Activity_windows_visible id: ActivityRecord{365b181a u0 com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity t4495 f} time:1015238
W/ActivityManager(  790): Activity pause timeout for ActivityRecord{3d4c9072 u0 com.nextcloud.client/com.owncloud.android.ui.activity.WhatsNewActivity t4495 f}
E/Diag_Lib(  312): linux_qmi_qmux_if_configure_port_timeout: mode=0, smd_timeout=20, hsic_timeout=60
E/Diag_Lib(  312): linux_qmi_qmux_if_configure_port_timeout: failed to open HSIC timeout config=/sys/devices/virtual/hsicctl/hsicctl0/modem_wait errno [2:No such file or directory]
E/Diag_Lib(  312): linux_qmi_qmux_io_open_conn: open SMD port /dev/smdcntl0 for RDWR
V/NotificationService(  790): pkg=de.mangelow.throughput canInterrupt=false intercept=true
V/NotificationService(  790): pkg=de.mangelow.throughput canInterrupt=false intercept=true
V/NotificationService(  790): pkg=de.mangelow.throughput canInterrupt=false intercept=true
D/TaskPersister(  790): removeObsoleteFile: deleting file=4492_task.xml
```

**Comment by stuart12** [comment]
Nextcloud dev 20170903 does not crash at startup on this tablet but I have not connected it to my server because I don't want to lose data.

**Comment by AndyScherzinger** [comment]
Hi @stuart12,

this isssue is solved for the next stable release 2.0.0. We will release 2.0.0RC6 probably next week so it should be to long until this is resolved.
